src/alfasim_sdk/_internal/units.py

```diff
@@ -158,3 +158,11 @@ def add_category_if_not_defined(*, category, quantity_type, valid_units):
         valid_units=std_volume_units,
     )
     # NOTE: "gas volume flow rate" and "gas volume" are also required, but they are already registered
+
+    angle_per_time_units = ["Hz", "rpm", "rad/s"]
+    db.AddCategory(
+        "angle per time",
```

Comment:
Is this to simplify conversions between those units?
I am asking because Frequency and Angular Velocity are a little bit different, even though they are interchangeable.

Reply:
Here I'm simply restricting the available units for category `angle per time`. With this change, we'll only show `["Hz", "rpm", "rad/s"]` to the user.

> I am asking because Frequency and Angular Velocity are a little bit different, even though they are interchangeable.

I'm guessing that this question is related to that fact that you saw `frequency` somewhere in the diff. That was incorrect, it should have been `category="angle per time"` and not `category="frequency"`.

Reply:
Actually, I was asking because usually `rad/s` is considered an angular velocity(`angle per time`), and `Hz` is considered a frequency (`inverse of time` or `1 per time`). We can covert one to another, but they are not the same.

I think you answered my question, it was made to simplify. Since they are interchangeable, we can consider the same category and make the conversions implicitly.

Reply:
> Here I'm simply restricting the available units...

Maybe we should add this to change log.